index.html

```diff
@@ -1,21 +1,148 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
 
-    <title>Responsive grid project</title>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- Add a link to your custom CSS here -->
+  <link rel="stylesheet" href="css/main.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Open+Sans:wght@300&display=swap"
+    rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Poppins:wght@600&display=swap"
+    rel="stylesheet" />
+  <title>Responsive grid project</title>
 </head>
 
 <body>
+  <!-- header -->
+  <header>
+    <nav>
+      <div class="box"></div>
+      <h1 class="title_top">
+        Maya Nelson </h1>
 
-    <!-- Add your markup here -->
+      <div class="ham_menu">
+        <div class="wrapper-menu">
+          <div class="line-menu half start"></div>
+          <div class="line-menu"></div>
+          <div class="line-menu half end"></div>
+        </div>
+      </div>
 
+      <p>PROJECT MANAGER</p>
+      <ul id="menu" class="menu">
+        <li><a href="index.html">ABOUT ME</a></li>
+        <li><a href="./css/resume.html">RESUME</a></li>
```

Comment:
your link looks to be pointing to the wrong location, this should just be resume.html